tests/test_back.js

```diff
@@ -410,18 +406,31 @@ tap.test('nockBack record tests', function (nw) {
     }
 
     nockBack(fixture, {afterRecord: afterRecord}, function (done) {
-      http.request(options).end();
-      done();
-
-      t.true(exists(fixtureLoc));
-
-      nockBack(fixture, function (done) {
-        t.true(this.scopes.length === 0);
-        done();
-
-        fs.unlinkSync(fixtureLoc);
-        t.end();
-      });
+        const server = http.createServer((request, response) => {
+            t.pass('server received a request')
+            response.writeHead(200);
+            response.end();
+            t.true(exists(fixtureLoc));
+
+        server.listen(() => {
+            const request = http.request({
+              host: 'localhost',
+              path: '/',
+              port: server.address().port
+            }, (response) => {
+              done()
+
+              t.is(200, response.statusCode)
+              t.true(exists(fixtureLoc))
+              t.true(this.scopes.length === 0)
+              fs.unlinkSync(fixtureLoc)
+
+              server.close(t.end)
+            })
+
+            request.on('error', t.error)
+            request.end()
+          })
```

Comment:
you are mixing up two spaces / four spaces indentation, which make is a bit hard to read, could you fix that please?

It looks like you forgot to close one of the brackets, see https://travis-ci.org/node-nock/nock/jobs/350166318#L1962

Make sure to run `npm test` locally and have it green before creating the pull request :)

Reply:
Ah, really sorry for this, my bad. Fixing :)

Perhaps I don't think the repo has a `.editorconfig` file, should we have one? @gr2m

Reply:
no please don’t, not everyone is using these. But you can add one to your home directory if you have a preferred style.

Usually Editors can refer the indentation style from the current file without additional configuration